package/MDAnalysis/topology/CRDParser.py

```diff
@@ -76,9 +72,9 @@ class CRDParser(TopologyReaderBase):
      - Resnums
      - Segids
 
-    Guesses the following attributes:
-     - Atomtypes
-     - Masses
+    .. versionchanged:: 2.8.0
+       Removed type and mass guessing (attributes guessing takes place now
+       through universe.guess_TopologyAttrs() API).
```

Comment:
For the sake of users who read this and may get confused about the word "removed", may I suggest the following ammendment (for here and elsewhere):
```suggestion
    .. versionchanged:: 2.8.0
       Type and mass are not longer guessed here. Until 3.0 these will still be
       set by default through through universe.guess_TopologyAttrs() API.
```

The intent here is to make sure that folks don't read these docs and go "well I don't have atomtypes and masses anymore".